cli/medperf/entities/schemas.py

```diff
@@ -0,0 +1,82 @@
+from datetime import datetime
+from pydantic import BaseModel, Field, validator, HttpUrl
+from typing import Optional, Union
+
+from medperf.enums import Status
+
+
+class MedperfSchema(BaseModel):
+    id: Optional[Union[str, int]]
+    name: str = Field(..., max_length=20)
+    owner: Optional[int]
+    created_at: Optional[datetime]
+    modified_at: Optional[datetime]
+    # TODO: This must change after allowing edits
+    state: str = "OPERATION"
+    is_valid: bool = True
```

Comment:
these should not exist for Result entity

Reply:
Created a DeployableSchema. check it out!